packages/react-core/src/components/Toolbar/__tests__/__snapshots__/Toolbar.test.tsx.snap

```diff
@@ -103,7 +103,7 @@ exports[`Toolbar should render with custom chip content 1`] = `
           class="pf-v5-c-toolbar__item pf-m-chip-group"
         >
           <div
-            aria-labelledby="pf-random-id-0"
+            aria-labelledby="generated-id"
```

Comment:
is this working correctly?  There are 2 elements with the same id in a page?

Reply:
That's just because it's using my GeneratedId mock which always returns this same string not the actual component.

If we need to have each of these unique in the snapshots I can update the mock to support that, but then we'll be back in the situation where snapshots often have to be updated just because another test was added before the snapshot test in a test suite.

Reply:
I think it is fine for snapshots to have duplicates.